version1/02 - Object types.md

```diff
@@ -90,863 +92,614 @@ The following notation is used in the tables below:
+| M450      | string    | kassasjonsvedtak                 |          |                 | kassasjonsvedtak  |  |  V   |  V   |   V   |      |      |        |                                                                                                                                 |
+| M453      | string    | kassasjonshjemmel                |          |                 | kassasjonshjemmel |  |  V   |  V   |   V   |      |      |        |                                                                                                                                 |
+| M451      | number    | bevaringstid                     |          |                 |                   |  |  V   |  V   |   V   |  V   |      |        |                                                                                                                                 |
+| M500      | string    | skjerming                        |          |                 | skjerming         |  |  V   |  V   |   V   |      |      |        |                                                                                                                                 |
+|           | timestamp | forventetKassasjonsDato          |          |                 |                   |  |  sm  |  V   |   V   |  V   |      |        |                                                                                                                                 |
+|           |           |                                  |          |                 |                   |  |      |      |       |      |      |        |                                                                                                                                 |
+| M711      | object    | virksomhetsspesifikkeMetadata    |          |                 |                   |  |  V   |  V   |       |      |      |        |                                                                                                                                 |
+|           |           |                                  |          |                 |                   |  |      |      |       |      |      |        |                                                                                                                                 |
+|           | ref       | refArkivdel                      |    c     |                 |                   |  |      |  V   |   V   |      |  V   |   V    | <p>&bull; exactly one of **refArkivdel** and **refForelderMappe** must be set</p>                                               |
+|           | ref       | refRegistrering                  |          |                 |                   |  |      |      |   V   |      |  V   |   V    | <p>&bull; references **Basisregistrering**</p><p>&bull; exactly one of **refBarnMappe** and **refRegistrering** must be set</p> |
```

Comment:
The comment no longer holds for the base type. I'd state in the comment that different files support different record types and urge readers to look at the other tables for details. You can then describe those details in the other tables:
* mappe: only supports basisregistrering
* saksmappe: supports all types, but moeteregistrering
* moetemappe: only supports moeteregistrering